embark-ui/src/components/TextEditorToolbar.js

```diff
@@ -4,18 +4,38 @@ import {Button, Nav, NavLink} from 'reactstrap';
 import classnames from 'classnames';
 import FontAwesomeIcon from 'react-fontawesome';
 
-const TextEditorToolbarTabs = {
-  Overview: 'overview',
-  Detail: 'detail',
-  Transactions: 'transactions',
-  Debugger: 'debugger',
-  Browser: 'browser'
+export const TextEditorToolbarTabs = {
+  Interact: 'Interact',
```

Comment:
I think we could have 1 object:
```{
interact: {name: 'Interact', icon: 'bolt'}
}```

what do you think?

Reply:
We can do that too. I just thought that, in terms of semantics, it'd make more sense to have "enum-like" constructs here. E.g. other components imports `TextEditorToolbarTabs` to make use of them, where icons might not be interesting at all.

If you prefer an object this way, I'll go ahead and change it :)

Reply:
The only reason I prefer an object is that it is easier to maintain.
Add/Remove/Rename a tab means changing only 1 place.